upload.py

```diff
@@ -28,6 +30,7 @@ def bayfiles_upload(files):
         resp = json.loads(r.content)
         if r.status_code == 200:
             print('BayFiles Uploaded: ' + resp['data']['file']['url']['short'])
+            progress_state.append(1)
```

Comment:
progress_state is used to check if everything is done. It will be used to delete the ZIP in the future